src/components/modules/MainPage/ShortCard/StyledShortCard.js

```diff
@@ -0,0 +1,101 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.div`
+    margin:0.5%;
+    border-radius: 40px;
+    display: flex;
+    flex-direction: column;
+    box-shadow: 0 3px 6px rgba(0, 0, 0, 0.4), 0 3px 6px rgba(0, 0, 0, 0.23); 
+    background-color:#C4C4C4;
+    width: ${({width}) => (width === "large" ? "90%" : "30%")};
+    @media (orientation: portrait){
+      width: 100%;
+      margin-bottom:10%;
+    }
+    
+`;
+
+export const CardHeader = styled.div`
+   border-radius: 40px;
+   padding: 30px 10px;
+   background-color: #59687B;
+   text-align: center;
+   color: white;
+   font-size: ${({cardSize})=> cardSize || "1.2rem"};
```

Comment:
this can be deleted as we don't use the prop cardSize, we can do it we find it applicable